packages/search/src/style.scss

```diff
@@ -198,6 +198,23 @@ $input-z-index: 20;
 			display: block;
 		}
 
+		// Removes the fade overlay when the search is open and focused
+		&.has-focus {
+			.search-component__input-fade {
+
+				&::before {
+					@include long-content-fade( $size: 0 );
```

Comment:
How about `display: none` instead of including a fade with `0` size?

Reply:
Seems like a better solution.